lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -8,10 +8,11 @@
         'moment-with-locales',
         'js/components/card/views/card',
         'teams/js/views/team_utils',
+        'edx-ui-toolkit/js/utils/html-utils',
         'text!teams/templates/team-membership-details.underscore',
```

Comment:
Note that these Underscore.js templates should also be reviewed.  This first one uses `<%=` in at least one location.